cli/builder/builder.go

```diff
@@ -87,20 +87,10 @@ func (cb *CLIBuilder) Build() (*cmdlib.Root, error) {
 	}
 
 	// pass in deps to build the root command
-	rootCmd := cmdlib.New(
-		cb.name,
-		cb.description,
-		cb.defaultRunHandler(logger),
-		clientCtx,
-	)
+	rootCmd := cmdlib.New(cb.name, cb.description, cb.defaultRunHandler(logger), clientCtx)
```

Comment:
nit: having these stuff on their own line usually simplifies diffs if there ever are

Reply:
yeah was left over from old changes. this file is gonna go soon anyways